README.md

```diff
@@ -46,7 +46,12 @@ mongo = Mongo::MongoClient.from_uri(mongo_uri)
 tailer = Mongoriver::Tailer.new([mongo], :existing)
 outlet = YourOutlet.new(your_params) # Your subclass of Mongoriver::AbstractOutlet here
 stream = Mongoriver::Stream.new(tailer, outlet)
+# install a shutdown handler so we can make sure to stop things cleanly
+trap('TERM') do
+  stream.stop
+end
 stream.run_forever(starting_timestamp)
+tailer.save_state
```

Comment:
These changes are upcoming for the tailer used in Simon as soon as this is approved / released.  I've tested that it respects SIGTERM locally, and shuts down cleanly.